pkg/config/config.go

```diff
@@ -95,8 +95,9 @@ func init() {
 	Datadog.SetDefault("disable_file_logging", false)
 	Datadog.SetDefault("syslog_uri", "")
 	Datadog.SetDefault("syslog_rfc", false)
-	Datadog.SetDefault("syslog_tls", false)
```

Comment:
We should remove `syslog_tls` from the configuration template (line 318) and add `syslog_tls_verify` instead.